widgets/src/main/resources/org/pentaho/gwt/widgets/themes/public/themes/ruby/globalRuby.css

```diff
@@ -597,15 +597,21 @@ div#promptPanel div.prompt-panel {
 }
 
 div#toppanel {
-  margin-top: 20px;
+  margin-top: unset;
 }
 
 div.parameter div.parameter-label {
   margin-bottom: 5px;
 }
 
 div#promptPanel div.parameter {
-  margin-bottom: 20px;
+  margin-bottom: unset;
+}
+
+.parameter-container > tbody > tr > td > div > table > tbody {
```

Comment:
If not possible in our limited time, you don't have to address my comments.

But the css selector` tbody > tr > td > div > table > tbody` seems really brittle. Could you use another css selector to target the html content?

I see other examples in the file such as`div#promptPanel div.parameter` and `div#promptPanel div.parameter`